src/System.Text.Primitives/System/Text/Encoding/Ascii/PrimitiveEncoder_ascii.cs

```diff
@@ -4,12 +4,12 @@
 
 using System.Runtime.CompilerServices;
 
-namespace System.Text
+namespace System.Text.Encodings
 {
-    public static partial class PrimitiveEncoder
+    public static partial class Ascii
     {
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static string DecodeAscii(this ReadOnlySpan<byte> bytes)
+        public static string ToUtf16String(this ReadOnlySpan<byte> bytes)
```

Comment:
This seems like an odd extension. This adds a ToUtf16String extension to any ROS<byte> instance, however, it will clearly only function if the data in that ROS<byte> happens to be ASCII. The naming (to me) would make me believe that this should also work for some other data. For example, I know my ROS<byte> is UTF-8 data, so I call this method thinking I will get a valid UTF-16 string.